locale/en_US/locale.po

```diff
@@ -71,8 +74,47 @@ msgstr "Primary Citation Format"
 msgid "plugins.generic.citationStyleLanguage.settings.citationFormatsPrimaryDescription"
 msgstr "Select which citation format you would like to display by default on your article landing page."
 
+msgid "plugins.generic.citationStyleLanguage.settings.citationFormatsPrimaryDescription.omp"
+msgstr "Select which citation format you would like to display by default on your book landing page."
+
 msgid "plugins.generic.citationStyleLanguage.settings.publisherLocation"
 msgstr "Publisher Location"
 
 msgid "plugins.generic.citationStyleLanguage.settings.publisherLocation.description"
 msgstr "Some citation formats request the geographic location of the publisher, such as \"London, U.K.\"."
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationUserGroups"
+msgstr "Author and Translator"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationUserGroupsDescription"
+msgstr "In some cases beside authors there also exists translators. Please choose a role for authors and translators."
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationUserGroups.omp"
+msgstr "Author, Editor and Translator"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationUserGroupsDescription.omp"
+msgstr "In some cases beside authors there also exist editors or translators. Please choose a role for authors, editors and translators."
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationChooseAuthor"
+msgstr "Author:"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationChooseEditor"
+msgstr "Editor:"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationChooseTranslator"
+msgstr "Translator:"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationChooseChapterAuthor"
+msgstr "Chapter Author:"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationOptionChooseAuthor"
+msgstr "Please choose a role for authors"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationOptionChooseEditor"
+msgstr "Please choose a role for editors"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationOptionChooseTranslator"
+msgstr "Please choose a role for translators"
+
+msgid "plugins.generic.citationStyleLanguage.settings.citationOptionChooseChapterAuthor"
+msgstr "Please choose a role for chapter authors"
```

Comment:
It is possible for for multiple user groups to be configured for each of these roles. I think we're going to need to adjust this to allow journals/presses to select multiple user groups for each role. Once changed, you can use the following text for each:

> Select the roles that should be identified as authors.